python-api/tests/integrationtests/baseclass.py

```diff
@@ -0,0 +1,412 @@
+"""
+Contains the Base class for integration tests.
+"""
+
+import sys
+import os
+import logging
+import tempfile
+import shlex
+import shutil
+try:
+    import subprocess32 as subprocess
+except ImportError:
+    import subprocess
+
+from glob import glob
+import numpy
+import scipy.misc
+import scipy.linalg
+import gi
+gi.require_version('Gst', '1.0')
+
+sys.path.insert(0, os.path.abspath(os.path.join(__file__, "../../../")))
+from gstswitch.server import Server
+from gstswitch.helpers import TestSources
+from gstswitch.controller import Controller
+from gi.repository import GLib
+from gstswitch.testsource import VideoSrc, AudioSrc
+
+# PATH = os.getenv("HOME") + '/gst/stage/bin/'
+PATH = '../tools/'
+
+
+class IntegrationTestbase(object):
+    """Base class for integration tests."""
+
+    # Tests are not allowed to have an __init__ method
+    def setup_method(self, _):
+        """Set up called automatically before every test_XXXX method."""
+        self.log = logging.getLogger()
+        self.log.setLevel(logging.DEBUG)
+        logging.basicConfig(
+            format="%(filename)s:%(lineno)d (%(funcName)s): %(message)s")
+
+        self.serv = None
+        self._sources = None
+        self.controller = None
+
+    def setup_server(self, record_file=False):
+        """Set up a gst-switch server for testing."""
+        assert self.serv is None
+
+        self.log.info("setting up Server")
+        self.serv = Server(path=PATH, video_format="debug",
+                           record_file=record_file)
+
+        self.log.info("running Server")
+        self.serv.run()
+        assert self.serv.is_alive()
+
+        self.log.info("waiting for Server to open Controller-Port")
+        self.serv.wait_for_output('tcp:host=0.0.0.0,port=5000')
```

Comment:
Does this string still match after I fixed the binding to all IPv4 and IPv6 addresses in https://github.com/timvideos/gst-switch/pull/206? I would have thought it would be `tcp:host=::,port=5000` now?